spec/mailers/message_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe MessageMailer, type: :mailer do
+  describe 'create' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.